protocol/app/prepare/prepare_proposal.go

```diff
@@ -71,8 +74,21 @@ func PrepareProposalHandler(
 			return &EmptyResponse, nil
 		}
 
+		var extCommitBzTx []byte
+		if len(req.Txs) >= slinkyabci.NumInjectedTxs {
+			extCommitBzTx = req.Txs[slinkyabci.OracleInfoIndex]
```

Comment:
It's not really determined in the dydx repo though--this is hard to tell here just because of the way I've had to separate out the PRs into manageable chunks. 

The [slinky prepare handler](https://github.com/skip-mev/slinky/blob/main/abci/proposals/proposals.go#L182) wraps the dydx handler. It injects the VE data into the `req.Txs` and then hands off execution to dydx so that we can construct the MarketPriceUpdates Tx with it. 

So the index at which the VEs are injected and the number of txs inject are both fixed in the Slinky impl. I can move the constants here, but changing them without also changing the upstream handler would just break. 

If it's important to be able to control this it can maybe be parametrized as inputs to the slinky prepare handler.

Reply:
You're saying that storing `OracleInfoIndex = 0` in the dYdX codebase would required changes to Slinky protocol code? It seems to me like this constant of 0 is fixed because VE is always at index 0

Reply:
Updated in `protocol/app/constants/constants.go`.

It is fixed--there's no other place it can be. Changing this constant in either place would simply break the handlers.